src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
@michaelficarra and @lydell I have made the decision to switch to babel which supports this currently; If any changes are needed in this PR i'd still be happy to make them but otherwise it can be closed.